backend/src/main/java/com/redhat/cloud/notifications/recipients/itservice/pojo/request/AccountRelationship.java

```diff
@@ -0,0 +1,37 @@
+package com.redhat.cloud.notifications.recipients.itservice.pojo.request;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "allOf",
+        "by"
+})
+public class AccountRelationship {
+
+    @JsonProperty("allOf")
+    private List<String> allOf = null;
+
+    @JsonProperty("by")
+    private By__1 by;
+
+    public List<String> getAllOf() {
+        return allOf;
+    }
+
+    public void setAllOf(List<String> allOf) {
+        this.allOf = allOf;
+    }
```

Comment:
General comment: if we only use this class and the other similar ones for deserialization of the IT response, `public` fields  will do the same job with a lot less code:
```
public class AccountRelationship {
    public List<String> allOf; // no need for '= null'
    public By__1 by;
}
```

Reply:
good catch. thx. The object structure is still on my list, but I wanted to make the tests green first.